src/main/groovy/io/seqera/wave/util/RegHelper.groovy

```diff
@@ -292,4 +292,9 @@ class RegHelper {
             log.debug "Unexpected error while closing http response - cause: ${e.message}", e
         }
     }
+
+    static boolean isValidImageName(String imageName){
+        final DOCKER_IMAGE_REGEX = /^(?:[a-zA-Z0-9-]+\.)?[a-zA-Z0-9-]+(?:\/[a-zA-Z0-9-]+)*$/
```

Comment:
Last one, see this description around separators characters within components of the image name:

```
The path consists of slash-separated components. Each component may contain lowercase letters, digits and separators. A separator is defined as a period, one or two underscores, or one or more hyphens. A component may not start or end with a separator.
```
can you make sure this is accounted for?

References:
- [Docker docs](https://docs.docker.com/reference/cli/docker/image/tag/#description)
- [OCI Distribution Spec](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pulling-manifests)

Reply:
to make it par with OCI specification, I am using regex from https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pulling-manifests

Reply:
so it should be good now

Reply:
thank you 👍